crates/macros/src/helpers.rs

```diff
@@ -26,3 +27,31 @@ impl GetDocs for &[Attribute] {
         get_docs(self)
     }
 }
+
+#[derive(Debug, Copy, Clone, FromMeta, Default)]
+pub enum RenameRule {
+    /// Methods won't be renamed.
+    #[darling(rename = "none")]
+    None,
+    /// Methods will be converted to camelCase.
+    #[darling(rename = "camelCase")]
+    #[default]
+    Camel,
+    /// Methods will be converted to snake_case.
+    #[darling(rename = "snake_case")]
+    Snake,
+}
+
+pub trait Rename {
+    fn renmae(self, rule: &RenameRule) -> Self;
+}
+
+impl Rename for String {
+    fn renmae(self, rule: &RenameRule) -> Self {
+        match *rule {
+            RenameRule::None => self,
+            RenameRule::Camel => ident_case::RenameRule::CamelCase.apply_to_field(self),
+            RenameRule::Snake => ident_case::RenameRule::SnakeCase.apply_to_field(self),
+        }
+    }
+}
```

Comment:
Just want to let you know that this was just changed in #413. You might wanna rebase onto master

Reply:
I see. I think do separate PR